### ISSUE TITLE
why voc2012test mp is so low(4%)?

### ISSUE THREAD

**Issue by zhangjiamin** [HiKapok/SSD.TensorFlow#45]
Hi, HiKapok:
Your code is so pretty, good job . I use voc2012test instead of voc2007test, but it's mp is so low, I don't understand the reason.

AP for pottedplant = 0.0000
AP for motorbike = 0.0000
AP for bottle = 0.0000
AP for cow = 0.0000
AP for tvmonitor = 0.0000
AP for person = 0.8002
AP for chair = 0.0008
AP for bus = 0.0000
AP for train = 0.0000
AP for sofa = 0.0000
AP for sheep = 0.0000
AP for boat = 0.0000
AP for horse = 0.0000
AP for diningtable = 0.0078
AP for cat = 0.0000
AP for bicycle = 0.0000
AP for aeroplane = 0.0000
AP for car = 0.0000
AP for dog = 0.0000
AP for bird = 0.0000
Mean AP = 0.0404

**Comment by HiKapok** [comment]
@zhangjiamin we have no labels for 2012-test set, you should evaluate online

**Comment by jamiechoi1995** [comment]
Hi all,

I evaluate the pretrained model provided by @HiKapok on VOC 2012test,
got a mAP=0.72760,

see: http://host.robots.ox.ac.uk:8080/anonymous/RCJ5AC.html

**Comment by HiKapok** [comment]
@zhangjiamin you should train another model using VOC07 Train + VOC07 Test + VOC12 Train, to match the official results, instead of using the provided model

**Comment by jamiechoi1995** [comment]
@HiKapok 
I think you @ a wrong person.

I notice this. This code actually implements the SSD* model, I will train it on 07++12 to test its performance.

**Comment by jamiechoi1995** [comment]
@HiKapok 

I think your code actually reproduce the SSD300 instead of SSD300* model,

the SSD* model refers to the model with **NEW** data augmentation, that is Section 3.6 Data Augmentation for Small Object Accuracy of the original paper which implements a "zoom out" operation that create mores small training examples.

The data augmentation in your code only implements random sample patch, horizontally flip and distortion, which refers to Section 2.2 of the original paper, and SSD is already trained with these augmentation in Section 2.2
![图片](https://user-images.githubusercontent.com/13494034/55536511-8a0eb100-56ec-11e9-9a2c-e4015aa7e969.png)

so your code can achieve a better result than SSD* when actually implements SSD , congratulations!

**Comment by HiKapok** [comment]
@jamiechoi1995 I'm sure the codes implements SSD300*

**Comment by jamiechoi1995** [comment]
@HiKapok 
I saw you define the zoom out function in 
https://github.com/HiKapok/SSD.TensorFlow/blob/b47ff6164c8925a8bbccc593719d5bbbab996058/preprocessing/ssd_preprocessing.py#L290

and use it in https://github.com/HiKapok/SSD.TensorFlow/blob/b47ff6164c8925a8bbccc593719d5bbbab996058/preprocessing/ssd_preprocessing.py#L364

**Comment by jamiechoi1995** [comment]
I use the default learning rate and steps to train SSD* on 07++12, I start from the VGG weights and trained for 120000 steps, my batchsize is 128, but I got **72.18** on VOC 2012 test, which is worse than the trained model you provided.
http://host.robots.ox.ac.uk:8080/anonymous/AILO4J.html
May I ask how did you trained the model you provided? How can I get similar results as in the original paper?
